acceptance/cmd/workspace/apps/run-local/script

```diff
@@ -0,0 +1,26 @@
+cd app
+
+# We first run the command with different entry poiny which starts unblocking script
+# so we don't need to start it in background. It will install the dependencies as part of the command
+trace $CLI apps run-local --prepare-environment --entry-point test.yml 2>&1 | grep -o "Hello, world"
+
+title "Starting the app in background..."
+$CLI apps run-local --prepare-environment --debug > ../out.run.txt 2>&1 &
+PID=$!
+
+cd ..
+
+echo Waiting for the app to start...
+sleep 7
```

Comment:
Could we come up with something to make this more reliable but also don't wait unnecessarily. Instead of sleeping, perhaps we can monitor out.run.txt for a certain message to appear that indicates that server is ready?

Reply:
Addressed in https://github.com/databricks/cli/pull/2555/commits/aaf77164fe66c4146ba7db8a8af081d030bd2a68